index.js

```diff
@@ -53,9 +56,11 @@ Cloneable.prototype.clone = function () {
   // the events added by the clone should not count
   // for starting the flow
   this.removeListener('newListener', onData)
+  this.removeListener('resume', onResume)
```

Comment:
As mentioned in the comment above, we're removing these listeners so events added by Clone does not start the flow. However, in practice, it seems that the intent here is not fulfilled.

`parent.pipe(this)` [here](https://github.com/mcollina/cloneable-readable/pull/33/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L110) emits adds a `data` listener which we successfully ignore since we've removed the listener, but it also emits a [`resume` event in a `nextTick`](https://github.com/nodejs/readable-stream/blob/master/lib/_stream_readable.js#L905).

We add back the `resume` on line 63 below but since resume is emitted in `nextTick`, we still act on it and reduce the `_clonesCount` erroneously.

I tried adding the event listener in a `nextTick`, but then that breaks case where user immediately adds a `data` event after cloning (The `clone sync` case from tests).

Reply:
To work around this issue, I've added a flag `_internalPipe` [extended `resume` method](https://github.com/mcollina/cloneable-readable/pull/33/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R50) and made it a noop when that flag is set. 
Let me know if this approach sounds sane enough.
It sure gets all tests green and let's us [delay the start](https://github.com/mcollina/cloneable-readable/pull/33/files#diff-13876b4beb64b9f156474dc78f9c923952a7ca210d4507b6b3135bbe244f8a60R93) 😄